src/task/style.css

```diff
@@ -258,3 +258,10 @@ footer {
   color: rgb(255, 92, 92);
   font-size: 0.75rem;
 }
+
+#assignee {
+  background-repeat: no-repeat;
+  background-position: right;
+  background-size: 42px;
```

Comment:
Can we use the shorthand [background](https://github.com/Real-Dev-Squad/website-dashboard/pull/193/files#diff-5a43e4e279e445a814167ecbef27ccf07a7f5ca55283e18504b81279b0f39270R263-R265) CSS property for all these?

Reply:
@shubham-y in new changes I am using background-position and background-size property only, and this time I used shorthand property as suggested by you